snmp_standard/mode/numericvalue.pm

```diff
@@ -191,18 +191,17 @@ sub check_data {
             $value = $value / $delta_time;
         }
     }
-    
+
     if ($options{entry}->{format_custom} ne '') {
         $value = eval "$value $options{entry}->{format_custom}";
     }
-    
-    my $exit = $self->{perfdata}->threshold_check(value => $value, 
+
+    my $exit = $self->{perfdata}->threshold_check(value => $value,
                                                   threshold => [ { label => 'critical-' . $options{num}, exit_litteral => 'critical' }, { label => 'warning-' . $options{num}, exit_litteral => 'warning' } ]);
     if (defined($options{entry}->{format_scale})) {
-        my ($value_mod, $value_unit) = $self->{perfdata}->change_bytes(value => $value);
-        if ($options{entry}->{format_scale_type} =~ /^network$/i) {
-            ($value_mod, $value_unit) = $self->{perfdata}->change_bytes(value => $value, network => 1);
-        }
+        my $network = $options{entry}->{format_scale_type} =~ /^network$/i ? { network => 1 } : undef;
```

Comment:
We can get an error with 'undef' in ```$network``` when you do: ```%$network```. Could you set by default: ```{}```

Reply:
Yep, fixed